src/ui/view/share/data/i18n/share_data_de.json

```diff
@@ -0,0 +1,3 @@
+{
+  "type-press-enter": "Tippen Sie und drücken Sie die <span>Eingabetaste</span>"
```

Comment:
I don't really like having HTML tags in translations. It's something we can discuss, but the idea of translations is to have something purely textual, even if the [library we use](https://react.i18next.com/latest/trans-component) allows for tag interpolation.

Reply:
We probably can split this translation in two separated translation, even though the first part of the translation left alone doesn't have that much sense. Using tags for translation is a common practice used in several libraries to isolate a part of it's content for styling. When used regularly (e.g. with `t` from `useTranslation`), it won't create any tag.

Reply:
We finally split the translation and removed the tag.